lib/utils/settings.py

```diff
@@ -49,7 +49,7 @@ def __init__(self):
         # We set create=False here because this code is executed on module load
         # and via imports also runs on generate-inx-files, which with the Nix
         # package manager is executed with a non-writable home directory.
-        user_dir = get_user_dir(create=False)
+        user_dir = get_user_dir(create=True)
```

Comment:
Why did you change this? This again broke the build on NixOS. See the comment above it for the reasoning.

Reply:
We got mutiple error message reports on this from users who tried the dev builds. But I think it's been fixed elsewhere. I honestly don't remember, I'll revert this change and will check again.